src/robot_interface/models/exceptions/robot_exceptions.py

```diff
@@ -202,6 +202,17 @@ def __init__(self, error_description: str) -> None:
     pass
 
 
+# An exception which should be thrown by the robot package if there is no new telemetry update.
+class RobotTelemetryNoUpdateException(RobotException):
```

Comment:
```suggestion
class RobotTelemetryNoUpdateException(RobotTelemetryException):
```

Reply:
It complains when adding that the RobotTelemetryException is trying to initianlize with its own error reason.
src/robot_interface/models/exceptions/robot_exceptions.py:185: note: "__init__" of "RobotTelemetryException" defined here
src/robot_interface/models/exceptions/robot_exceptions.py:209: error: Unexpected keyword argument "error_reason" for "__init__" of "RobotTelemetryException"  [call-arg]

Reply:
I see